### DISCUSSION TITLE
[Script request] Bookstack

### DISCUSSION THREAD

**Discussion by CrazyWolf13** [community-scripts/ProxmoxVE#30] [Request script] [answered]
### Application Name

Bookstack

### Website

https://www.bookstackapp.com/

### Description

BookStack is a simple, self-hosted, easy-to-use platform for organising and storing information.

### Due Diligence

- [X] I have searched existing [scripts](https://community-scripts.github.io/Proxmox/scripts) and found no duplicates.
- [X] I have searched existing [discussions](https://github.com/community-scripts/ProxmoxVE/discussions?discussions_q=) and found no duplicate requests.

**Comment by MickLesk** [comment, marked as answer]
I create an script for this. 
#Blocked

**Comment by MickLesk** [comment]
Bookstack added:
  bash -c "$(wget -qLO - https://github.com/community-scripts/ProxmoxVE/raw/main/ct/bookstack.sh)"

**Reply by CrazyWolf13** [reply]
Thanks insanely much!!!
